feature/experimental/tunnel/otg_tests/tunnel_interface_based_resize_test/README.md

```diff
@@ -0,0 +1,97 @@
+ *   The test should send 2000 IPv4 and 2000 1Pv6 flows from the ATE:Port1 to the DUT:Port1. These flows should equally be split between the 2x /24 IPv4 and 2x /64 IPv6 destination prefixes mentioned in Subtest-1 bullet#4 above. The flows should use variations in the following,
+        * Source IP address
+        * Source Port
+        * Destination IP adresses.
+        * Destination Port
+        * Payload protocol
+ *   Incoming traffic on DUT:PORT1 should be load shared across all 64 Tunnel interfaces for encapsulation and then routed out the combination of lag and singleton interfaces, i.e. lag1, lag2 and Port6.
+ *   Follow all the "verifications" steps in [*] below
+
+## Subtest-3: Reduce number of tunnel interfaces 
+ *   Start with the state in Subtest-2 above with active traffic flow.
+ *   Reduce the number of Tunnel interfaces, e.g. from 32 to 16 for each of the tunnel destination group by address-family:
+        * If the static routes are used to forward traffic to tunnel, please disable or delete the static routes in this test to simulate the reduction in available paths.
+ *   Incoming traffic on DUT:PORT1 should now be load shared across all remaining 16 Tunnel interfaces per tunnel-destination by address-family.
+ *   Follow all the verifications steps in [*] below
+ *   Inflight reduction in number of tunnel interfaces shouuldn't cause traffic drop.
+
+## Subtest-4: Restore the number of tunnel interfaces 
+ *   Start from the condition in Subtest-3 above with inflight traffic from OTG:Port1.
+ *   Increase number of Tunnel interfaces, e.g. From 16 to 32 per tunnel-destination by address-family.
+ *   Incoming traffic on DUT:PORT1 should now be load shared across all 32 Tunnel interfaces per tunnel-destination by address-family. Follow all the verifications steps in [*] below
+ *   Inflight increase in the number of tunnel interfaces and related static route shouldn’t cause traffic drops.
+
+#### [*] Verification steps
+ *   Verify the next hop counters for packets being diverted or sent for encapsulation. Test should fail if drops recorded.
+ *   Verify the tunnel interfaces counters to confirm traffic encapsulation. Test should fail if the counts dont match the original count of packets sent by OTG
+ *   Verify the tunnel interfaces for optimal load balancing of traffic/flow. Acceptable deviation is <=3%
+ *   Verify the tunnel interfaces for optimal load balancing of traffic/flow. Acceptable deviation is <=3%. This is true for tunnel destinations of both address families.
+ *   Validate system for:
+        * Health-1.1. checks.
+        
+
+    
+## Config Parameter coverage
+
+TODO: OpenConfig definition required for Tunnel protocol under interfaces/interfaces/interface/tunnel/ as GRE, IP-IP, GUE etc. 
+ *   interfaces/interfaces/interface/tunnel/config/dest/
+ *   interfaces/interfaces/interface/tunnel/config/src/
+ *   interfaces/interfaces/interface/tunnel/ipv4/addresses/address/config/ip
```

Comment:
Should this also cover?
/interfaces/interface/tunnel/config/ttl 
/interfaces/interface/tunnel/config/gre-key

Reply:
- /interfaces/interface/tunnel/config/gre-key
   The goal of gre-key was to tie flows to the same tunnel interface in both directions. That is not the requirement in Google. 
- /interfaces/interface/tunnel/config/ttl
We dont set it. Not a requirement either. By default TTL is copied from inner header